<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDialog</class>
 <widget class="QDialog" name="HelpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="gv_contents">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>601</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tw_guide">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tb_generalguide">
    <attribute name="title">
     <string>Interface</string>
    </attribute>
    <widget class="QListWidget" name="lw_contentsitems">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>561</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>(1)Quick Sketching Interface Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(1.1)Quick Sketching Interface Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(1.2)Quick Sketching Interface Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(2)Quick Flow Diagnostics Interface Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(2.1)Quick Flow Diagnostics Interface Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(2.2)Quick Flow Diagnostics Interface Manual</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tb_specificguide">
    <attribute name="title">
     <string>Contents</string>
    </attribute>
    <widget class="QListWidget" name="lw_userguide">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>531</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>(1). How to Get the Surfaces from Sketch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(2). How to Get the Surfaces from a File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(3.1)How to Edit the User Input Parameters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(3.2)How to Edit the User Input Parameters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(4.1) User Input Form</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(4.2) User Input Form</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(4.3) User Input Form</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(4.4) User Input Form</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(5)Mesh Generation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(6)Flow Diagnostics</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(7)Pore Volume</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(8)CrossSection Feature</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(9)Export Options</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
